raster-foundry/raster-foundry
# Scale color map to histogram on first preview in lab

## Overview

Scale color map to histogram on first preview in lab.

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~


### Notes
It still acts weird if you open the histogram then preview, but we decided that it is not worth fixing until all the AST stuff is done.

## Testing Instructions

 * Open a tool run 
* Enter all inputs with a landsat 8 band, hit apply
* without opening a histogram, click preview and verify that the color map is scaled to the correct range.

Closes #2529
